services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -457,4 +513,20 @@ class SwitchRulesSpec extends BaseSpecification {
 
         return rules
     }
+
+    void checkTrafficCountersInRules(FlowEndpointPayload flowEndpoint, trafficThroughIngressRule = true) {
+        def rules = northbound.getSwitchRules(flowEndpoint.datapath).flowEntries
+        def ingressRule = filterRules(rules, flowEndpoint.portNumber, flowEndpoint.vlanId, null)[0]
+        def egressRule = filterRules(rules, null, null, flowEndpoint.portNumber).find {
+            it.instructions.applyActions.fieldAction.fieldValue == flowEndpoint.vlanId.toString()
+        }
+
+        assert ingressRule.flags.contains("RESET_COUNTS")
+        assert (trafficThroughIngressRule ? ingressRule.packetCount > 0 : ingressRule.packetCount == 0)
```

Comment:
It can actually be simplified to
assert trafficThroughIngressRule == (ingressRule.packetCount > 0)

Reply:
Thanks, fixed.